--- origsrc/gentoo-0.15.1/src/cmdgrab.c	2009-03-07 16:45:05.000000000 -0600
+++ src/gentoo-0.15.1/src/cmdgrab.c	2009-03-08 17:05:36.859375000 -0500
@@ -24,6 +24,13 @@
 
 #include "cmdgrab.h"
 
+#ifdef stdout
+#undef stdout
+#endif
+#ifdef stderr
+#undef stderr
+#endif
+
 /* ----------------------------------------------------------------------------------------- */
 
 #define	GRAB_CHUNK	(512)		/* We read this many bytes on each "input available" callback. */
