--- origsrc/gentoo-0.11.56/src/Makefile.am	2004-12-26 18:55:36.000000000 -0600
+++ src/gentoo-0.11.56/src/Makefile.am	2006-08-27 18:27:30.073125000 -0500
@@ -199,7 +199,7 @@
 
 localedir=${datadir}/locale
 
-CFLAGS=@GENTOO_CFLAGS@ @GTK_CFLAGS@ -I../widgets \
+CFLAGS=@GENTOO_CFLAGS@ @GTK_CFLAGS@ -I$(top_srcdir)/widgets \
   -DPATH_CFG=\"$(sysconfdir)\" \
   -DPATH_GRC=\"$$HOME/\" \
   -DPATH_ICN=\"$(datadir)/gentoo/icons\"\
--- origsrc/gentoo-0.11.56/widgets/Makefile.am	2003-10-12 08:45:23.000000000 -0500
+++ src/gentoo-0.11.56/widgets/Makefile.am	2006-08-27 18:27:30.088750000 -0500
@@ -7,7 +7,7 @@
 modules=odemilbutton.c  odlabel.c  odscrolledbox.c
 
 # Targets ---------------------------------------------------------------------
-CFLAGS=@CFLAGS@ @GTK_CFLAGS@ -I../src
+CFLAGS=@CFLAGS@ @GTK_CFLAGS@ -I$(top_srcdir)/src
 
 noinst_LIBRARIES=libwidgets.a
 libwidgets_a_SOURCES=$(headers) $(modules)
